### PR TITLE
Add dd/mm/yyyy date format support

### PR DESCRIPTION
**Pull request by tephenavies**
### What this PR does

Add back support for dd/mm/yyyy common Australian date format to version 8.

This is a description of what I've done in this PR, especially explaining choices made to make the reviewer's job easier. If I don't replace this paragraph with an actual description, my PR will probably be ignored.

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: lib/Table/TableColumn.ts [hunk 1]
```diff
@@ -158,47 +158,163 @@ export default class TableColumn {
...
+      const d = +dayString;
+      const m = +monthString;
+      const y = +yearString;
+      if (Number.isInteger(d) && Number.isInteger(m) && Number.isInteger(y)) {
+        if (d > 31 || m > 12 || y > 9999) {
+          parsingFailed = true;
+          return null;
         }
+        return new Date(centuryFix(y), m - 1, d);
+        return new Date(`${centuryFix(y)}/${m}/${d}`);
```
Comment (nf-s) [inline lib/Table/TableColumn.ts]: There are two returns here

Comment (nf-s) [reply lib/Table/TableColumn.ts]: and omg months start at 0 - that is very upsetting

**Review by nf-s — CHANGES_REQUESTED** [review]
Looks good, just needs that extra return statement removed.

At some point we should bring in missing date parsing from master - https://github.com/TerriaJS/terriajs/blob/6153d332aa3bd160ddbbd968be37a339a4f6ad96/lib/Map/TableColumn.js#L485

For example `2020-Q1` type format

File: lib/Table/TableColumn.ts (hunk 1 above)
Comment (tephenavies) [reply lib/Table/TableColumn.ts]: ikr

Comment (tephenavies) [inline lib/Table/TableColumn.ts]:
```suggestion
```

**Review by nf-s — APPROVED** [review]

**Pull request merged**